apps/meteor/app/apps/client/orchestrator.ts

```diff
@@ -107,15 +107,10 @@ class AppClientOrchestrator {
 		}
 
 		return (result as App[]).map((app: App) => {
-			const { latest, price, pricingPlans, purchaseType, isEnterpriseOnly, modifiedAt, bundledIn } = app;
+			const { latest, ...rest } = app;
 			return {
 				...latest,
-				price,
-				pricingPlans,
-				purchaseType,
-				isEnterpriseOnly,
-				modifiedAt,
-				bundledIn,
+				...rest,
```

Comment:
IIRC `latest` has some properties with the same name as the ones in the top-level `app` object, so spreading the `rest` here will likely override those if done this way.

I also like extracting exactly the info we need from the original `app` object so that we don't "overshare" unnecessary data, but that is not objectively better (unlike the above), just my two cents :P

Reply:
Got it, will solve this shortly. I think we should take the refactoring of this app object to the engineering day. Right now it is a simple spread but it can become a big problem in the future.

Reply:
Solved